css/templates/land-see/page.css

```diff
@@ -0,0 +1,281 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../core';
+@import '../../../components/byline/byline';
+@import '../../../components/carousel/carousel';
+@import '../../../components/dropdown/dropdown';
+@import '../../../components/footer/footer';
+@import '../../../components/images/fullpage-hero';
+@import '../../../components/images/images';
+@import '../../../components/inputs/inputs';
+@import '../../../components/navbar/navbar';
+@import '../../../components/related-article/related-article';
+@import '../../../components/related-section/related-section';
+@import '../../../components/socialbox/socialbox';
+@import '../../../components/social-follow/social-follow';
+@import 'page-vars';
+
+h1,
+.h1,
+h2,
+.h2 {
+  letter-spacing: 0;
+}
+
+h3,
+.h3 {
+  color: var(--color-font-primary);
+  font-family: var(--font-secondary);
+  letter-spacing: .6px;
+}
+
+h4,
+.h4 {
+  letter-spacing: .2px;
+  line-height: var(--line-height-2);
+}
+
+.ampstart-headerbar {
+  background: var(--color-primary);
+  color: var(--color-font-primary);
+  padding-right: var(--h5);
+}
+
+.ampstart-headerbar-nav {
+  flex: none;
+}
+
+@media (--breakpoint-md) {
+  .ampstart-headerbar-nav {
+    flex: 1;
+  }
+  .ampstart-headerbar-fixed-link {
+    margin-right: var(--h5);
+  }
+}
+
+.ampstart-headerbar-nav > ul {
+  text-align: right;
+  float: right;
+}
+
+.ampstart-headerbar-title {
+  font: 600 var(--h4)/var(--line-height-5) var(--font-secondary);
+}
+
+.ampstart-navbar-trigger {
+  font-size: 1.5rem;
+}
+
+.ampstart-headerbar-fixed-link {
+  margin-right: 0;
+}
+
+.ampstart-sidebar .ampstart-nav-dropdown section header {
+  height: 0;
+  width: 0;
+  margin: 0;
+}
+
+.ampstart-headerbar .ampstart-nav-dropdown {
+  display: none;
+}
+
+.ampstart-nav-item {
+  font-family: var(--font-navigation);
+  text-transform: uppercase;
+  letter-spacing: .1rem;
+}
+
+.sidebar-nav-item {
```

Comment:
Can you namespace this class so that it is sepcific to your template (i think you use land-see)